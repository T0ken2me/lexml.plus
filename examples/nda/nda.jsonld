{
  "@context": "https://lexml.plus/vocab/0.1/vocab.jsonld",
  "@id": "https://lexml.plus/rule/NDA-RETURN-30D",
  "@type": "ObligationRule",
  "subject": "Recipient",
  "action": "ReturnOrDestroyCI",
  "appliesFrom": "2025-09-01",
  "timeframeISO8601": "P30D",
  "wasDerivedFrom": "contract://nda/2025-01#return"
}