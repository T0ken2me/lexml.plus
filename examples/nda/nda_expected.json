{
  "meta": {
    "assessment_date": "2025-09-05",
    "contract_id": "urn:contract:NDA:Demo",
    "jurisdiction": "LU",
    "engine_version": "lexml.assess.v0.1"
  },
  "results": [
    {
      "rule_id": "NDA-RETURN-30D",
      "status": "pending",
      "deadline": "2025-09-30",
      "evaluated_on": "2025-09-05",
      "explanation": "Recipient must return or destroy within 30 days of termination.",
      "sources": [
        "house:nda.return.v3"
      ]
    }
  ]
}